packages/fxa-settings/src/components/Banner/index.stories.tsx

```diff
@@ -44,3 +44,12 @@ export const DismissibleInfo = () => (
     <Subject />
   </AppLayout>
 );
+export const FancySuccess = () => (
```

Comment:
The reason this is included here is because I thought I was going to be implementing the UX in Figma for confirm signup code and wanted to see what this looked like, but that'll be done in a separate ticket.